### ISSUE TITLE
Odd behaviour with budgie-desktop-git 

### ISSUE THREAD

**Issue by sparkylein** [solus-project/budgie-desktop#506]
Couldn't wait for the newest budgie version so cloned, compiled and installed it.

Observed the following oddity (don't know if it's intentional, please forgive if it is):
Action: Mouse-button-click on the menu icon
Behavior: Menu appears
Action: without clicking on an item move the mouse cursor over to the panel over the battery or Bluetooth icon
Behavior: the popup window of the respective icons appear (like a right-click) and the menu disappears
Action: without clicking anything move the cursor over the menu icon
Behavior: the menu appears

Hope you guys can see the same behavior and decide if it's a bug or not.
Just wanted to mention what I observed before Solus 1.2.1 goes GOLD :P 

sparky

**Comment by ikeydoherty** [comment]
What's the oddity? It's very hard to follow this - the popovers are designed to work like a menubar, in that if you have a popover active, rolling over any other widget that activates a popover will then move to that popover, much like in other popover-based desktops

**Comment by sparkylein** [comment]
So does it mean that as long an item in a menu or an icon on a panel is not clicked they stay all "active" by design?
If this is the case then please disregard me mentioning it.

**Comment by ikeydoherty** [comment]
You clicked one popover first right, and then rolled over to another without clicking to lose focus?

**Comment by sparkylein** [comment]
yes.

**Comment by sparkylein** [comment]
Although the other popovers appear...

**Comment by ikeydoherty** [comment]
So it behaves like a menubar, as its supposed to? :) I don't see a usability issue with this and it's always done this (Well, Budgie 10 series has)

**Comment by sparkylein** [comment]
No problem, if you play with it and feel it's the right behavior, all is good.
Sorry for bothering you.

While I have you here: I changed the height of the menu to be the almost screen height. It serves me better as it utilizes more real estate. If I go about and add a selector in the menu settings to change the height from "normal" to "full height" would you welcome my involvement?
And how do I go about? Committing to github and then you decide if you take this on or not?

**Comment by ikeydoherty** [comment]
Full height? Can you should me what you mean?

**Comment by sparkylein** [comment]
In file BudgieMenuWindow.vala line 345 I changed this:
set_size_request(300, 510);
to this
set_size_request(300, (Gdk.Screen.height() - 30)); // maybe a lower value would work as well
but I think to make it user selectable would be a good thing.

**Comment by sparkylein** [comment]
In the forum I've even presented the idea to change the menu from a popover to a sliding window like raven (just from the other side...). Looked at the code but realized for this more is needed than changing a few lines of code...

**Comment by ikeydoherty** [comment]
No I mean I'd like to see a screenshot :p

**Comment by sparkylein** [comment]
![2016-07-21-145129_1280x800_scrot](https://cloud.githubusercontent.com/assets/20585812/17039951/ef1a8aa6-4f51-11e6-9e29-93ab07335bee.png)

Does this work for you?

**Comment by ikeydoherty** [comment]
Yeah this personally is not something I want to see in Budgie by default.
With that said, if you want a _proper_ full screen launcher applet, we do have a public API which will enable this :)

**Comment by sparkylein** [comment]
Should have told you, I have a strange taste... :P
Will look into the API (suppose I find it in the wiki) 

I close this issue as solved so I don't bother you any longer. 
Take care.
